### DISCUSSION TITLE
DirectoryTree右侧是否可以添加下拉菜单

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [ant-design/ant-design#44112] [Q&A]
DirectoryTree右侧是否可以添加下拉菜单啊？

<img width="838" alt="image" src="https://github.com/ant-design/ant-design/assets/4521375/16eedb16-bba0-415e-b5d7-2419170d566a">

<img width="326" alt="image" src="https://github.com/ant-design/ant-design/assets/4521375/e8a3e8a6-6c29-43b9-b479-41f73bf4ab87">

**Comment by afc163** [comment]
好主意。

**Reply by jiangxiaoqiang** [reply]
那就是目前还不能是吧，将近10年前有人提过https://github.com/ant-design/ant-design/issues/5151

**Reply by jiangxiaoqiang** [reply]
我好像从issue里找到方案了
